packages/addons/addon-depends/icu/package.mk

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="icu"
+PKG_VERSION="67-1"
+PKG_SHA256="571d3723a8055d85e513b19b70b4f7f370da10df084c3bb598e0fb8333da0a16"
+PKG_LICENSE="https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE"
+PKG_SITE="http://www.icu-project.org"
+PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION}.tar.gz"
```

Comment:
```suggestion
PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION//./-}.tar.gz"
```

Reply:
Isn't it simpler to use PKG_VERSION="67-1"?

Reply:
yes it would, but the version is 67.1, just the download+tag is 67-1 for odd reasons

Reply:
Done